parsec/core/backend_connection/cmds.py

```diff
@@ -326,7 +380,11 @@ async def _send_cmd(transport: Transport, serializer, **req) -> dict:
                 raise BackendProtocolError("Invalid request data according to backend")
 
             # Backward compatibility with older backends (<= v2.3)
-            if rep.status == "invalid_certification" and "timestamp" in rep.reason:
+            if (
+                rep.status == "invalid_certification"
+                and rep.reason is not None
```

Comment:
Oh, so you found a bug here?

Reply:
`reason` attribute is an `Optional[str]` is it an error?